src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3685,6 +3685,9 @@
   <data name="InvalidOperation_EditFailed" xml:space="preserve">
     <value>The assembly update failed.</value>
   </data>
+  <data name="NotSupported_DebuggerAttached" xml:space="preserve">
+    <value>Assembly updates not supported while debuggger is attached.</value>
```

Comment:
```suggestion
    <value>Assembly updates cannot be applied while a debugger is attached.</value>
```

Reply:
Yes, this change was about the other block.